frontend-instructor/src/components/analytics/Analytics.jsx

```
@@ -0,0 +1,73 @@
+          <div className='w-1/3 p-4 border-2 border-gray-300 rounded-xl flex justify-between items-center'>
+            <h1 className='text-xl'>Total Likes</h1>
+            <h1 className='text-xl'>{courseData?.totalLikes} </h1>
+          </div>
+          <div className='w-1/3 p-4 border-2 border-gray-300 rounded-xl flex justify-between items-center mx-4'>
+            <h1 className='text-xl'>Total Dislikes</h1>
+            <h1 className='text-xl'>{courseData?.totalDisLikes} </h1>
+          </div>
+          <div className='w-1/3 p-4 border-2 border-gray-300 rounded-xl flex justify-between items-center'>
+            <h1 className='text-xl'>Total Revenue</h1>
+            <h1 className='text-xl'>
+              {parseFloat(courseData?.price) *
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)